components/disabled_button/README.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Disabled buttons are greyed out to indicate to users that they cannot undertake the action associated with it.
+title: Disabled Button
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
+> Last Updated: March 28, 2019
+
+# Disabled buttons
+
+A button that can’t be clicked and doesn’t react to hover. A disabled button is greyed out to indicate to users that they cannot undertake the action associated with it. This is usually for page logic reasons.
+
+## Visual examples
+
+<component-preview path="components/disabled_button/sample.html" height="100px" width="800px"> </component-preview>
+
+## Design Guidance
+
+* Disabled buttons block users from accomplishing their task without telling them what is wrong. Rather than disabling a button, keep it enabled and show an error message why they can’t proceed. 
+* Before using a disabled button, make sure your research proves it makes the user interface easier to understand.
```

Comment:
Let's put this bullet point first, the bullet above then provides a rationale for this